### ISSUE TITLE
Super lazy version url incorrect

### ISSUE THREAD

**Issue by Mingyu-Kim** [jabelone/OpenCV-for-Pi#15]
in readme.md shell file URL in the super lazy version needs to be changed to
https://raw.githubusercontent.com/jabelone/OpenCV-for-Pi/master/lazy_install.sh
Otherwise it will throw redirect error

**Comment by abhiTronix** [comment]
@Mingyu-Kim  https://github.com/abhiTronix/TBB_Raspberry_pi

**Comment by xurizaemon** [comment]
NB: The link posted above by @abhiTronix is to a Github project with no downloads. `abhiTronix/TBB_Raspberry_pi` links to `abhiTronix/OpenCV_Raspberry_pi_TBB` where [Files](https://github.com/abhiTronix/OpenCV_Raspberry_pi_TBB#files) currently says,

> Latest .deb files and full support is only provided through email (redacted@example.com) . Consider supporting my countless hours of hardwork and helping me out by making small Donation for my ongoing Independent A.I. Research. Thankyou.

So for people looking for an installable, open source build of OpenCV, that's not the project you want.

**Comment by abhiTronix** [comment]
@xurizaemon Yes, if you're looking for an opensource build of OpenCV,  That repo might not suitable for you,  but I may update it soon with **OpenCV .deb files(unoptimized versions)** as a demo.
### Still, I want to make a few things clear:

- If you ever mailed to the email given at repo, _I always encourage to build files by yourself first as it might help you to understand OpenCV more correctly and you might able to customize it properly according to your needs._
- The files I'm providing is the _**optimized custom build** of OpenCV, Which is hardware specific and will be built according to your current system settings_ and also support Latest **TBB**(_for ultimate performance_). This enables maximized performance possible plus **I will be helping you if at any instance installation goes wrong or any future problems**. So the super lazy version is for **_anyone who wanted maximum performance with minimum effort_**. Yes, little donation for that is a lot to ask but as an independent hobbyist, it will do.

Hope, This might help you. Goodluck
